sdk/version.go

```diff
@@ -53,8 +51,24 @@ func GetVersion(dir string) (ver string, err error) {
 	return
 }
 
+func GetVersion() (string, error) {
```

Comment:
Remove, this is just duplicating `GetLatestVersion` which seems like the better name anyway.